src/api/config/clock.rb

```diff
@@ -90,4 +90,9 @@ module Clockwork
   every(1.week, 'consistency check', at: 'Sunday 03:00') do
     Old::ConsistencyCheckJob.perform_later
   end
+
+  # Expire assignments after 24h
+  every(1.hour, 'expire assignments') do
```

Comment:
I would increase the interval for this task to one day. I don't see much value in being so precise, particularly in this case.

Reply:
Fine for me